### ISSUE TITLE
Error: Cannot find module 'firmata'

### ISSUE THREAD

**Issue by dasamepu** [rwaldron/johnny-five#1777]
Hello, I was trying to run the example but when I write in the command-line "node app.js" it shows me "Error: Cannot find module 'firmata'".

I'm not sure but I think it's because the file "C:\Users\user\AppData\Local\Temp\build123.tmp\StandardFirmata.cpp.hex" doesn't exist in my PC and in the Aduino IDE console log at bottom it shows me this file: "C:\Users\user\AppData\Local\Temp\arduino_build_560749/StandardFirmata.ino.elf".

Maybe that is the reason beacause is not working or not, but I'll be really thankfull if you help me.
![Error_Cannot_find_module_firmata](https://user-images.githubusercontent.com/87206494/152699473-98959c7d-b40b-4952-920d-12da81fcc822.png)

**Comment by dtex** [comment]
That particular error message isn't related to the Arduino code, it's just saying it can't find the node.js module "firmata". That should have been installed when you installed johnny-five so it's a bit of a mystery. Anything non-standard about your install process? Were there any errors when you ran `npm i johnny-five`? I assume this is running on Windows and connected to an Arduino via USB correct? If so try `npm i firmata` from within the `proteusnode` directory.

**Comment by dasamepu** [comment]
Yes, when I install johnny-five it looks like it wasn't intalled properly.
![npm_install_johnny-five](https://user-images.githubusercontent.com/87206494/152728039-c5559cdb-c62d-48ad-a188-f822e2eb3d4d.png)

And when I install firmata it also shows me errors and says that I have should have to install python, but why if I won't use python?
![npm_intall_firmata_1](https://user-images.githubusercontent.com/87206494/152731773-5c6150b8-0ec2-45d4-a04d-1ea69a1236e3.png)
![npm_intall_firmata_2](https://user-images.githubusercontent.com/87206494/152731801-a4d9d2db-e496-4e39-8c3a-5e92bc3be781.png)

**Comment by Amaranthusss** [comment]
I have [the same problem](https://github.com/rwaldron/johnny-five/issues/1786), at this moment could only put manually `firmata` library data to `node_modules` and it helps.

**Comment by WyattSanders** [comment]
> I have [the same problem](https://github.com/rwaldron/johnny-five/issues/1786), at this moment could only put manually `firmata` library data to `node_modules` and it helps.

I've found the solution here:
https://github.com/rwaldron/johnny-five/issues/1807#issuecomment-1809561715
